spec/Section 2 -- Language.md

```diff
@@ -1095,6 +1095,11 @@ and operations.
 As future versions of GraphQL adopt new configurable execution capabilities,
 they may be exposed via directives.
 
+Note: When defining a directive, it is recommended to namespace the directive name 
+to prevent name collisions with directives added in future versions of the specification. 
+For example, `@fb_auth` directive would represent an authentication directive 
+with the prefix `fb` as namespace.
```

Comment:
Thus, we are talking about the **prefix**, and not about the **namespace**. GraphQL has no concept of namespaces. Perhaps the term _namespace_ is not worth mentioning at all?

Reply:
What other term would you use then?

I didn’t want to only use the term  « prefix » because it focuses on the how rather than what we are trying to achieve. One can use any other technique like suffixing if they prefer, to achieve « namespacing ».

Reply:
I say that the term _namespace_ should not be used, because  as a rule a certain meaning is invested in it. In addition, some syntax rules are always associated with namespaces. There is nothing here. Despite the fact that someone may not like the term prefix (I don't like it either), it is described here - without clear syntax rules.

Reply:
Agreed, especially since there's been past discussion about adding namespaces as a first class concept to GraphQL. We didn't follow that path, but we both want to avoid confusion and make sure that in some rare case we revisit namespaces that we don't accidentally create an ambiguity here.